data/publiccloud/terraform/azure.tf

```
@@ -0,0 +1,154 @@
+# Configure the Microsoft Azure Provider
+provider "azurerm" {}
+
+variable "count" {
+ default = "1"
+}
+
+variable "name" {
+ default = "openqa-vm"
+}
+
+variable "type" {
+ default = "Standard_A2_v2"
+}
+
+variable "region" {
+ default = "westeurope"
+}
+
+variable "image_id" {
+ default = "SLES12-SP4-Azure-BYOS.x86_64-0.9.0-Build3.51.vhd"
```

in my understanding this value always will be outdated , so maybe we should clear show this ? 
Two options : 
1. totally remove default 
2. if first is not possible put some template there  
```suggestion
 default = "SLES<>-Azure-BYOS.x86_64-0.9.0-Build3.51.vhd"
```
```suggestion
 default = "SLES<VERSION>-Azure-BYOS.<ARCH>-0.9.0-Build<BUILD_NUMBER>.vhd"
```

---

I went for the option 1, as I did for gce.tf initially. Image should be a mandatory argument, as it might change almost every day/week.